tests/tests.py

```diff
@@ -230,6 +229,7 @@ def test_single_form_csv_with_rep_fields_start_with_repeat(self):
         for line1, line2 in izip(open(get_filename('models.py'),'r'),open(get_filename(cmp_fileName),'r')):
             self.assertEqual(line1,line2)
     def test_single_form_csv_with_rep_fields_many_nested(self):
```

Comment:
If the whole method were commented out then setup.py would try to call a method that no longer exists, requiring that two files are changed and that two such comments would be made. perhaps it would be better to print a message describing the situation in the method so that future users would be reminded that something is missing?